### PR TITLE
Feat(Core): Added extra cooldown for wirebugs

### PR DESCRIPTION
**Pull request by WetU**

### CODE REVIEW

File: HunterPie.Integrations/Datasources/MonsterHunterRise/Entity/Player/MHRWirebug.cs [hunk 1]
```diff
@@ -14,6 +14,7 @@ public sealed class MHRWirebug : IEventDispatcher, IUpdatable<MHRWirebugExtrasSt
     private double _maxTimer;
     private double _cooldown;
     private double _maxCooldown;
+    private double _maxExtraCooldown = 0.0;
```
Comment (Haato3o) [inline HunterPie.Integrations/Datasources/MonsterHunterRise/Entity/Player/MHRWirebug.cs:17]: Unnecessary field as it's only being used inside the `Update` method, that can be a local variable.
```suggestion

```

File: HunterPie.Integrations/Datasources/MonsterHunterRise/Entity/Player/MHRWirebug.cs [hunk 2]
```diff
@@ -153,17 +154,18 @@ private set
...
     public void Update(MHRWirebugData data)
     {
         IsAvailable = data.IsAvailable;
         IsTemporary = data.IsTemporary;
-        MaxCooldown = data.Structure.MaxCooldown;
-        Cooldown = data.Structure.Cooldown;
         WirebugState = data.WirebugState;
+        Cooldown = data.Structure.Cooldown + data.Structure.ExtraCooldown;
+        _maxExtraCooldown = data.Structure.ExtraCooldown > 0.0f ? Math.Max(_maxExtraCooldown, data.Structure.ExtraCooldown) : 0.0;
+        MaxCooldown = Cooldown > 0.0 ? Math.Max(MaxCooldown, data.Structure.MaxCooldown + _maxExtraCooldown) : 0.0;
```
Comment (Haato3o) [inline HunterPie.Integrations/Datasources/MonsterHunterRise/Entity/Player/MHRWirebug.cs:168]:
```suggestion
        var maxExtraCooldown = data.Structure.ExtraCooldown > 0.0f ? Math.Max(_maxExtraCooldown, data.Structure.ExtraCooldown) : 0.0;
        MaxCooldown = Cooldown > 0.0 ? Math.Max(MaxCooldown, data.Structure.MaxCooldown + maxExtraCooldown) : 0.0;
```

**Review by Haato3o — APPROVED** [review]
LGTM!

**Comment by WetU** [comment]
Changes before:
before

After:
after

**Pull request merged**